### PR TITLE
feat: add table column and fix sorting bug

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue [hunk 1]
```diff
@@ -239,6 +244,7 @@ watch(() => state.selectedStatus, (selectedStatus) => {
                              :page-size.sync="state.pageSize"
                              row-cursor-pointer
                              sortable
+                             :sort-by="state.sortBy"
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:247]: issue: The `state.sortDesc` is not currently bound. However, if you're not planning to provide a default value, wouldn't it be unnecessary to bind `state.sortDesc` to the table? Could `state.sortDesc` also be considered unnecessary in this case?

**Review by WANZARGEN — COMMENTED** [review]
I like your clean code :) 
I've added some comments on areas that might need further review.

File: apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:247]: Oh, I see! I'll remove it then!

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by WANZARGEN — COMMENTED** [review]
It appears there might have been a misunderstanding regarding my previous comments. If `state.sortDesc` is bound to the table or if the value of `options` received as an argument in the event handler is directly used to update the `queryHelper`, then it implies that `state.sortDesc` is unnecessary.

Given that 'created_at' is set as the default value for `state.sortBy`, it does seem appropriate for `state.sortBy` to be bound.

**Pull request merged**